kalite/caching/tests/cache_tests.py

```diff
@@ -13,35 +13,35 @@
 from .. import caching
 from fle_utils.django_utils import call_command_with_output
 from kalite.testing.base import KALiteTestCase
-from kalite.topic_tools import get_video_cache
+from kalite.topic_tools import get_content_cache
 
 
 class CachingTest(KALiteTestCase):
-    video_cache = get_video_cache()
+    content_cache = get_content_cache()
```

Comment:
So if I remember correctly, `get_content_cache` will fetch videos too, not just mp3s and pdfs?